tb_plugin/tensorboard_plugin_torch_profiler/static/trace_embedding.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2014 The Chromium Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
```

Comment:
Any potential issue with this, given that our LICENSE file is most likely different?

Reply:
This file is *copy* and *modified* by https://github.com/catapult-project/catapult.
And `pytorch/kineto` has the same license. 

I'm not familiar enough with these terms. Would you mind confirm could we remove the copyright statement?(IMO, that's OK).

Thanks.